### ISSUE TITLE
xpdf fails with Error: Unresolved inheritance operation

### ISSUE THREAD

**Issue by proski** [Homebrew/homebrew-x11#216]
All packages are installed with default configs. I'm running up-to-date Mac OS X El Capitan 10.11.5 with up-to-date XQuartz 2.7.9.
```
$ xpdf -v
xpdf version 3.04
Copyright 1996-2014 Glyph & Cog, LLC
$ xpdf
Error: Unresolved inheritance operation
$ brew doctor
Your system is ready to brew.
$ otool -L /usr/local/bin/xpdf
/usr/local/bin/xpdf:
	/usr/local/opt/freetype/lib/libfreetype.6.dylib (compatibility version 19.0.0, current version 19.3.0)
	/usr/lib/libz.1.dylib (compatibility version 1.0.0, current version 1.2.5)
	/usr/local/opt/openmotif/lib/libXm.4.dylib (compatibility version 5.0.0, current version 5.4.0)
	/opt/X11/lib/libXt.7.dylib (compatibility version 8.0.0, current version 8.0.0)
	/opt/X11/lib/libXp.6.dylib (compatibility version 9.0.0, current version 9.0.0)
	/opt/X11/lib/libXext.6.dylib (compatibility version 11.0.0, current version 11.0.0)
	/opt/X11/lib/libXpm.4.dylib (compatibility version 16.0.0, current version 16.0.0)
	/opt/X11/lib/libSM.6.dylib (compatibility version 7.0.0, current version 7.1.0)
	/opt/X11/lib/libICE.6.dylib (compatibility version 10.0.0, current version 10.0.0)
	/opt/X11/lib/libX11.6.dylib (compatibility version 10.0.0, current version 10.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1226.10.1)
	/usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 120.1.0)
```
To debug this issue, I installed rxvt-unicode, and found that it runs properly (i.e. opens a window).

**Comment by dunn** [comment]
Does this happen if xpdf is built outside of Homebrew?

**Comment by proski** [comment]
I built unpatched xpdf 3.0.4 on the same system. I had to supply an option to `configure` to find FreeType:

    ./configure --with-freetype2-includes=/usr/local/include/freetype2

Once I built the sources, I ran `xpdf/xpdf`, and it reported `Error: Unresolved inheritance operation`

So the problem is present in the upstream version.

**Comment by dunn** [comment]
If you could report it to the developers, that would be great!

**Comment by proski** [comment]
After I reinstalled openmotif with `brew install --build-from-source`, I get another error
```
$ xpdf
Warning: Widget must be a VendorShell.
Error: attempt to add non-widget child "dsm" to parent "xpdf" which supports only widgets
```
gdb shows the error happens in XPDFViewer::initToolbar()
```
pageNumText = XmCreateTextField(toolBar, "pageNum", args, n);
```
The error is well known in MacPorts: https://trac.macports.org/ticket/19383
The fix appears to be to recompile libXt with -Wl,-flat_namespace
https://trac.macports.org/changeset/47999

I'm not sure I can report that upstream in a meaningful way. It's some misconfiguration. Why did the error change after I recompiled openmotif? Is openmotif wrong to use conflicting names? Then why is the fix in libXt?
